q2_boots/core_metrics.py

```diff
@@ -32,24 +33,71 @@ def core_metrics(ctx, table, sampling_depth, metadata,
 
     tables = bootstrapped_tables.values()
 
+    alpha_metric_names = alpha_metrics
```

Comment:
Maybe make the internal vars for alpha and beta metrics have different names from the kwargs. . . idk exactly what would be good names though, but the lines aliasing them kinda throw me off and would be confusing if you missed them.

Maybe call the one you create below `alpha_metrics_actions` or something?